docker-compose.yaml

```diff
@@ -39,6 +39,8 @@ services:
     restart: always
     expose:
     - "8080"
+    ports:
```

Comment:
There's a security reason we don't expose these ports in the base docker-compose file - we can add another overlay docker-compose file to expose the ports to be used in local development.